projects/m3/003-roman-numerals/solutions/enricozoboli/solution_roman_numerals.py

```diff
@@ -0,0 +1,30 @@
+def roman_calc(s):
+    """
+    Takes a roman number from the user and return the sum in a decimal form 
+    assuming that a correct number is entered by roman standars. 
+    If the first element of the string is lesser than the second the function
+    process the subtraction, add the difference to the sum of all the 
+    following elements and subtract the value of the second element which is 
+    added in the recursion back from the base case. If the first element is 
+    grater than the second the function simply add every element's values.
+    """
+    roman_values = {
```

Comment:
When you use functions in general, don't define constant variables within the function, but outside them, in order to not have multiple useless assignments

Reply:
Thanks for your suggestion. You are absolutely right.